src/main/python/ayab/plugins/ayab_plugin/ayab_control.py

```diff
@@ -181,88 +202,83 @@ def __cnfLine(self, lineNumber):
             # adjust lineNumber with current block
             lineNumber += BLOCK_LENGTH * self.__lineBlock
 
-            imgHeight = self._image.imgHeight()
-        
-            # work out which line of data to send
-            imgRow, color, indexToSend, sendBlankLine, lastLine = self._knit_func(lineNumber, imgHeight)
-            
-            # create bitarray
+            # get data for next line of knitting
+            imgRow, color, indexToSend, sendBlankLine, lastLine = self.__knit_func(lineNumber)
```

Comment:
Trying to knit singlebed yields: `TypeError: _singlebed() missing 1 required positional argument: 'lineNumber'`.
Possible to add a unit test for cnf_line?

Reply:
Don't we already have tests for `cnf_line()` in `test_communication.py` and `test_communication_mockup.py`?